src/SDKs/NotificationHubs/NotificationHubs.Tests/Tests/ScenarioTests.NotificationHubTests.CRUD.cs

```diff
@@ -52,8 +52,28 @@ public void NotificationHubCreateGetUpdateDelete()
                 Assert.Equal("Succeeded", getNamespaceResponse.ProvisioningState, StringComparer.CurrentCultureIgnoreCase);
                 Assert.Equal("Active", getNamespaceResponse.Status, StringComparer.CurrentCultureIgnoreCase);
 
+                var createParameter = new NotificationHubCreateOrUpdateParameters()
+                {
+                    Location = location,
+                    WnsCredential = new WnsCredential()
+                    {
+                        PackageSid = "ms-app://s-1-15-2-1817505189-427745171-3213743798-2985869298-800724128-1004923984-4143860699",
+                        SecretKey = "w7TBprR-9tJxn9mUOdK4PPHLCAzSYFhp",
```

Comment:
@amolr this will get flagged when you try to publish. Any reason this has to be hardcoded?